myuw/templates/handlebars/card/accounts/library.html

```diff
@@ -52,19 +50,27 @@ <h4 class="card-badge-label">You owe</h4>
                         </div>
                         <div class="pull-right">
                             <span class="card-badge-value text-danger">${{ formatPrice fines }}</span>
-			    &nbsp;
-                            <span class="card-badge-action pull-right"><a href="https://p.lib.washington.edu/payfines/" target="_blank" data-linklabel="Pay Library Fees">Pay fees</a></span>
+			  
+                            <span class="card-badge-action"><a href="https://p.lib.washington.edu/payfines/" target="_blank" data-linklabel="Pay Library Fees">Pay fees</a></span>
                         </div>
                     </div>
 		</li>
                 {{/if}}
+                <li>
```

Comment:
I don't think this should be a list item, as it's the CTA and semantically it's own element on the card (vs. one of the statuses, which are within a list).